bin/heat-flow-module-L0/9_plate_cooling_model_of_lithosphere.ipynb

```diff
@@ -0,0 +1,257 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Now, let's plot the heat flow due to the plate model with max ltuhisphere
</pre><p>lithosphere is misspelled at the end of the sentence</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Oh no, this was wrong on so many levels :D Thanks for catching this!